guides/source/form_helpers.md

```diff
@@ -54,10 +54,10 @@ One of the most basic forms you see on the web is a search form. This form conta
 To create this form you will use `form_with`, `label_tag`, `text_field_tag`, and `submit_tag`, respectively. Like this:
 
 ```erb
-<%= form_with(url: "/search", method: "get") do %>
-  <%= label_tag(:q, "Search for:") %>
-  <%= text_field_tag(:q) %>
-  <%= submit_tag("Search") %>
+<%= form_with url: "/search", method: "get" do %>
+  <%= label_tag :q, "Search for:" %>
+  <%= text_field_tag :q %>
+  <%= submit_tag "Search" %>
```

Comment:
I'd say `method: :get`, and use the `form.` appropriate helpers instead of *_tag.